### DISCUSSION TITLE
Scheme "dev" not found from repository! Please reconfigure your project.

### DISCUSSION THREAD

**Discussion by sandrews-inc** [codemagic-ci-cd/codemagic-docs#2636] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Run a build on my Flutter app with `--flavor dev` as a build option.

I confirmed that both the .xcodeproj and the .xcworkspace specify the scheme.
<img width="143" alt="Screenshot 2024-07-16 at 11 33 21 AM" src="https://github.com/user-attachments/assets/d3d3780d-176b-47ca-a11c-0763e46cfa1d">

<img width="263" alt="Screenshot 2024-07-16 at 11 33 14 AM" src="https://github.com/user-attachments/assets/52af7502-4bff-493b-854a-cced05eef752">

### Expected results

The build works locally and finds the scheme without issue.

### Actual results

<img width="863" alt="Screenshot 2024-07-16 at 11 35 04 AM" src="https://github.com/user-attachments/assets/21da9bd8-4573-41b7-8165-9044299c6690">

### Build id (optional)

66968bed4d7f12809d2a4f48

**Comment by dtrdic** [comment]
Hello @sandrews-inc, schematic names are case-sensitive. Can you please double-check if your scheme is written in different capitalization?

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.

**Comment by mfrischbutter** [comment]
I had the same issue since doing this migration: https://docs.flutter.dev/packages-and-plugins/swift-package-manager/for-app-developers

Locally is everything working, but codemagic can't find the scheme.

I reverted everything back like written in the migration guide and everything works fine again. I don't really need the swift package manager, i've just tested some things.

I just wanted to mention this in case someone else is looking for the same problem. No need to reopen the ticket for me.

**Comment by artebiakinOleksandr** [comment]
I had the same issue. Everything worked fine locally, but not in the CI/CD environment.
After adding `flutter pub get` to the **Post-clone script** section, it started working.

**Reply by DFelten** [reply]
Same issue. I had to add this step manually so that I could build via Codemagic.
